src/site/antora/modules/ROOT/pages/log4j-jul.adoc

```diff
@@ -15,116 +15,175 @@ Licensed to the Apache Software Foundation (ASF) under one or more
-| {log4j2-url}/javadoc/log4j-api/org/apache/logging/log4j/Level.html#OFF[`OFF`]
+[#bridge-handler]
+== Using `Log4jBridgeHandler`
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#SEVERE[`SEVERE`]
-| {log4j2-url}/javadoc/log4j-api/org/apache/logging/log4j/Level.html#ERROR[`ERROR`]
+.Are you a Spring Boot user?
+[TIP]
+====
+Spring Boot will automatically configure `Log4jBridgeHandler`.
+====
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#WARNING[`WARNING`]
-| {log4j2-url}/javadoc/log4j-api/org/apache/logging/log4j/Level.html#WARN[`WARN`]
+If setting the `java.util.logging.manager` system property is not possible, the JUL-to-Log4j bridge offers an implementation of JUL's
+https://docs.oracle.com/javase/{java-target-version}/docs/api/java/util/logging/Handler.html[`Handler`]
+abstract class, which redirects all log events to Log4j Core:
+`org.apache.logging.log4j.jul.Log4jBridgeHandler`.
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#OFF[`OFF`]
-| {log4j2-url}/javadoc/log4j-api/org/apache/logging/log4j/Level.html#INFO[`INFO`]
+[WARNING]
+====
+The `Log4jBridgeHandler` requires Log4j Core as logging implementation and will fail with other Log4j API implementations.
+====
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#CONFIG[`CONFIG`]
-| custom `CONFIG` level
+In order to use `Log4jBridgeHandler` you can either:
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#FINE[`FINE`]
-| {log4j2-url}/javadoc/log4j-api/org/apache/logging/log4j/Level.html#DEBUG[`DEBUG`]
+* modify the default JUL configuration file `logging.properties` to only contain:
++
+[source,properties]
+----
+# Set Log4jBridgeHandler as only handler for all JUL loggers
+handlers = org.apache.logging.log4j.jul.Log4jBridgeHandler
+----
++
+See the
+https://docs.oracle.com/javase/8/docs/api/java/util/logging/LogManager.html[JRE documentation]
+for details about the format and location of the `logging.properties` file.
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#FINER[`FINER`]
-| {log4j2-url}/javadoc/log4j-api/org/apache/logging/log4j/Level.html#TRACE[`TRACE`]
+* or call the `Log4jBridgeHandler.install()` method in your code.
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#FINEST[`FINEST`]
-| custom `FINEST` level
+[IMPORTANT]
+====
+Usage of `Log4jBridgeHandler` introduces a considerably higher overhead that the usage of `LogManager`, since logging events need to traverse the entire JUL logging pipeline followed by the logging pipeline of the Log4j API implementation.
```

Comment:
```suggestion
Usage of `Log4jBridgeHandler` introduces a considerably higher overhead, since logging events need to traverse the entire JUL logging pipeline followed by the logging pipeline of the Log4j API implementation.
```